### PR TITLE
USHIFT-4069: rebase fix to workaround MCO's use of go templates in yaml files

### PR DESCRIPTION
**Pull request by copejon**

### CODE REVIEW

**Comment by copejon** [comment]
test-rebase failed during builder image pull
/retest

**Comment by copejon** [comment]
Same thing:
`Failed pulling builder image., the build src-amd64 failed after 47s with reason PullBuilderImageFailed: Failed pulling builder image.]}`
/retest

File: scripts/auto-rebase/rebase.sh [hunk 1]
```diff
@@ -565,6 +565,12 @@ update_openshift_manifests() {
     title "Modifying OpenShift manifests"
 
     #-- Kubelet -------------------------------------------
+
+    # The tlsCipherSuites field was change from a scalar value to gotemplate which broke the yaml formatting and
+    # confused yq. Before processing, delete the offending go templated field.
+    # https://github.com/openshift/machine-config-operator/commit/3b979e1ddf2a6e2c3e9b4a7872e31db888da1d57
+    sed -n -i '/tlsCipherSuites:/,/{{- end }}/d' "${REPOROOT}/assets/core/kubelet.yaml"
```
Comment (pmtk) [inline scripts/auto-rebase/rebase.sh:572]: I guess we can delete following line that's ~20-30 lines below?
```
    yq -i 'del(.tlsCipherSuites)' "${REPOROOT}/assets/core/kubelet.yaml"
```

**Pull request merged**